apps/src/StudioApp.js

```diff
@@ -3165,23 +3120,20 @@ StudioApp.prototype.displayAlert = function(
   }
   var renderElement = container[0];
 
-  var handleAlertClose = function() {
-    ReactDOM.unmountComponentAtNode(renderElement);
-  };
-  ReactDOM.render(
-    <Alert
-      onClose={handleAlertClose}
-      type={props.type}
-      sideMargin={props.sideMargin}
-      closeDelayMillis={props.closeDelayMillis}
-      childPadding={props.childPadding}
-    >
-      {alertContents}
-    </Alert>,
-    renderElement
+  const playspaceAlert = React.createElement(
+    Alert,
+    {
+      onClose: () => {
+        ReactDOM.unmountComponentAtNode(renderElement);
+      },
+      type: type,
+      sideMargin: type === 'notification' ? undefined : 20,
+      closeDelayMillis: type === 'notification' ? 5000 : undefined,
+      childPadding: type === 'notification' ? '8px 14px' : undefined
```

Comment:
NIT (up to you):
This may end up being cleaner to do something like this:
```
let alertProps = {
  onClose: ...,
  type: type
};
if (type === 'notification') {
  alertProps.closeDelayMillis = 5000;
  alertProps.childPadding = '8px 14px';
} else {
  alertProps.sideMargin = 20;
}
...
const playspaceAlert = React.createElement(Alert, alertProps, alertContents);
```